cmds/truncate/truncate.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Truncate - shrink or extend the size of a file to the specified size
+//
+// Synopsis:
+//     truncate [OPTIONS] [FILE]...
+//
+// Options:
+//     -s: size in bytes
+//     -c: do not create any files
+//
+// Author:
+//     Roland Kammerer <dev.rck@gmail.com>
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var (
+	flags struct {
+		s int64
+		c bool
+	}
+	cmd = "truncate [-c] -s size file..."
+)
+
+func init() {
+	defUsage := flag.Usage
+	flag.Usage = func() {
+		os.Args[0] = cmd
+		defUsage()
+	}
+	flag.BoolVar(&flags.c, "c", false, "Do not create files.")
+	flag.Int64Var(&flags.s, "s", -1, "Size in bytes.")
+	flag.Parse()
```

Comment:
flag.Parse should always be in main (I know you took this from rm, it's wrong there, too)